packages/browser/src/backend.ts

```diff
@@ -12,7 +12,9 @@ const sendRavenEvent = Raven._sendProcessedPayload.bind(Raven) as SendMethod;
 
 /** Normalizes the event so it is consistent with our domain interface. */
 function normalizeRavenEvent(event: SentryEvent): SentryEvent {
-  const ex = (event.exception || {}) as { values?: SentryException[] };
+  const ex = ((event && event.exception) || {}) as {
```

Comment:
There was a issue when running it with SDK loader that`event` was null/undefined.

Reply:
That's not to be solved here.

Reply:
Can we fix this in captureEvent and change the top-level signature to `SentryEvent | undefined`? This is way too deep and also defies type checking.